dts/common/nordic/nrf54h20.dtsi

```diff
@@ -68,6 +68,56 @@
 				nordic,tasks-mask = <0xfffffff0>;
 			};
 		};
+
+		cpuflpr: cpu@e {
+			compatible = "nordic,vpr";
+			reg = <14>;
+			device_type = "cpu";
+			clock-frequency = <DT_FREQ_M(320)>;
+			riscv,isa = "rv32emc";
+			nordic,bus-width = <64>;
+
+			cpuflpr_vevif_rx: mailbox {
+				compatible = "nordic,nrf-vevif-task-rx";
+				status = "disabled";
+				interrupt-parent = <&cpuflpr_clic>;
+				interrupts = <0 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <1 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <2 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <3 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <4 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <5 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <6 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <7 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <8 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <9 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <10 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <11 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <12 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <13 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <14 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <15 NRF_DEFAULT_IRQ_PRIORITY>,
```

Comment:
I think those should be removed since tasks mask is `0xffff0000`. But this can be done in a separate PR.

Reply:
I should check if the code is index-sensitive, if it is, we'd need to change more stuff (like eg use interrupt-names).

Reply:
I think it should work, L15 has similar case:
https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/riscv/nordic/nrf54l15_cpuflpr.dtsi#L26-L40